public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/response_parser.ts

```diff
@@ -222,3 +148,60 @@ export default class ResponseParser {
     return dateTime(dateTimeValue).valueOf();
   }
 }
+
+const PARAM_RE = /([\w\W]+):([\w]+)(?:\s?=\s?([\w\W]+))?/;
```

Comment:
is there a way to rename this re to something that says what it is we are matching for? param and re are just very common you know?